charts/kubescape-operator/values.yaml

```diff
@@ -772,6 +772,11 @@ prometheusExporter:
     targetPort: 8080
     protocol: TCP
 
+  env: []
```

Comment:
I would prefer to have a variable enableWorkloadMetrics that would add ENABLE_WORKLOAD_METRICS in the deployment

Reply:
Sounds good to me too. Please take a look and let me know if I've missed anything.